app/models/subscribe_link.rb

```diff
@@ -0,0 +1,180 @@
+class SubscribeLink < ApplicationRecord
+  PLATFORMS = %w[apple spotify overcast pocketcasts youtube youtube_feed amazon antenna anytime apollo breez castamatic castbox castro curiocaster deezer fountain global goodpods gpodder hark iheart lnbeats luminary moon pandora player podbean podaddict podapp podguru_apple podguru_feed podrepublic podfriend podlp_apple podlp_guid podstation podurama podverse snipd sonnet steno truefans tunein].freeze
+
+  PLATFORM_HREFS = {
```

Comment:
pulled directly from the [Podcast Index proposal](https://github.com/nathangathright/podcast-platform-links?tab=readme-ov-file#show-links)